[32mINFO    [0m DeleteLogger:test_CourierDelete.py:23 Testing courier deletion without ID
[35mDEBUG   [0m DeleteLogger:test_CourierDelete.py:25 Status code 404 was returned and message{'code': 404, 'message': 'Not Found.'}